enhancements/cluster-logging/forwarding-json-structured-logs.md

```diff
@@ -128,154 +101,104 @@ New *pipeline* field:
 For each log entry; if `parse: json` is set _and_ the entry is valid JSON, the output record will include a `structured` field _equivalent_ to the JSON entry.
 It may differ in field order and use of white-space.
 
-All record will have a `message` field for backwards compatibility.
+### Output type elasticsearch
 
-#### Output  configuration for type:elasticsearch
+For most output types it is sufficient to enable `parse: json` to forward JSON data.
```

Comment:
@alanconway Which option is correct? parse: json or structured: json

Reply:
`parse: json` is correct. Thanks for spotting this - I left a couple of `structured: json` behind by mistake. Fixed.